app/views/admin/worldwide_organisations_translations/index.html.erb

```diff
@@ -69,12 +69,12 @@
         <%= render "govuk_publishing_components/components/button", {
           text: "Next",
           margin_bottom: 6,
-          # data_attributes: {
-          #   module: "gem-track-click",
-          #   "track-category": "form-button",
-          #   "track-action": "financial-report-button",
-          #   "track-label": "Create new report"
-          # }
+          data_attributes: {
```

Comment:
are these data-attributes in line with what Ruth is expecting? There is no way of knowing that this action is from the worldwide organisations translations page. Should the "track-action" be "worldwide-organisation-translation-button"?

Also same Q on the track-label as per the button text; should this be "Create new translation"

Reply:
I don’t believe this was tagged at all in the current implementation but I think you are right that the `track-action` attribute should relate to the fact that this is the translations page so I’m changing that as suggested. The `track-label` attribute should  match the button text so I will leave that as it is.